frontend/src/components/atoms/icons/filled/Email.tsx

```diff
@@ -0,0 +1,34 @@
+interface EmailProps {
+  color: string;
+  backgroundColor: string;
+  onClick?: () => void;
+}
+
+function Email(props: EmailProps) {
+  return (
+    <div
+      className={
+        props.backgroundColor +
+        " " +
+        "w-8 h-8 rounded-lg inline-flex justify-center items-center"
+      }
```

Comment:
Could you use `clsx` here?
https://www.npmjs.com/package/clsx

`className={clsx("w-8 h-8 ....", props.backgroundColor)}`